packages/xfce4_terminal.rb

```diff
@@ -36,5 +37,13 @@ def self.build
 
   def self.install
     system "make install DESTDIR=#{CREW_DEST_DIR}"
+    system "cat <<'EOF'> xfce4-terminal
+WAYLAND_DISPLAY=wayland-0
+GDK_BACKEND=wayland
+DISPLAY=
+#{CREW_PREFIX}/bin/xfce4_terminal $@
```

Comment:
Probably should put quotes around `\"$@\"`.

Reply:
No need, as `'EOF'` will escape all meaning of syntax automatically

Reply:
I'm talking about putting quotes around the arguments, not escaping the characters.

Reply:
Done